app/questionnaire/rules.py

```diff
@@ -30,20 +30,25 @@ def evaluate_goto(goto_rule, metadata, answer_store, group_instance):
     :return:
     """
     if 'when' in goto_rule.keys():
-        when = goto_rule['when']
+        all_conditions_met = True
 
-        if 'id' in when:
-            answer_index = when['id']
-            filtered = answer_store.filter(answer_id=answer_index, group_instance=group_instance)
-            if len(filtered) == 1:
-                return evaluate_rule(when, filtered[0]['value'])
+        for when_rule in goto_rule['when']:
+            condition_met = False
 
-        elif 'meta' in when:
-            key = when['meta']
-            value = get_metadata_value(metadata, key)
-            return evaluate_rule(when, value)
+            if 'id' in when_rule:
+                answer_index = when_rule['id']
+                filtered = answer_store.filter(answer_id=answer_index, group_instance=group_instance)
+                if len(filtered) == 1:
```

Comment:
Should this raise an exception or at least log an error as an `else` to this; not sure what is expected if more than one answer comes back (can that happen for multiple answer_instances?) but currently it will silently return False which could be a nasty one to debug. Another solution could be to get answer_store.filter to take an `expected_match_count=1` and have it do the log/exception so anyone calling filter doesn't need to deal with it.